lib/util/http/types.ts

```diff
@@ -23,3 +23,7 @@ export interface RequestStats {
   duration: number;
   queueDuration: number;
 }
+
+export interface OutgoingHttpHeaders extends Record<string, unknown> {
+  [header: string]: string | string[] | undefined;
```

Comment:
Remove and add this types to second record type

Reply:
Replaced by `Record<string, string | string[] | undefined>`, because empty interfaces don't lint well

Reply:
Better use a type def then?